### PR TITLE
Added contributor Apk23

### PR DESCRIPTION
**Pull request by Apk23**
Added myself as a contributor 😃

### CODE REVIEW

File: src/data/contributors/AtharvaKhedkar.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "Atharva Khedkar",
+  "skills": ["Python ", "Flutter", "Machine Learning"],
+  "cause": "I code for Passion",
```
Comment (Abhishek-kumar09) [inline src/data/contributors/AtharvaKhedkar.json:4]: `I code for` is already the part of string, `passion` would be enough.

Comment (Apk23) [reply src/data/contributors/AtharvaKhedkar.json]: Made the necessary changes

**Comment by Apk23** [comment]
Updated with changes, ready to merge

**Comment by Abhishek-kumar09** [comment]
Welcome Atharva! have a great way ahead with the community. Feel free to explore more about it.

**Comment by Apk23** [comment]
Thanks😄

**Pull request merged**